### DISCUSSION TITLE
Add additional date formats

### DISCUSSION THREAD

**Discussion by nf9k** [rendercv/rendercv#134] [Ideas]
From runtime output: Please use either YYYY-MM-DD, YYYY-MM, or YYYY format.  

I'm from the US and was trying to use MM-DD-YYYY.

Perhaps consider allowing two digit years?  It'll be a while before Y3k.

**Comment by sinaatalay** [comment]
Thank you for opening an issue. I don't think we need to add new date formats because the dates in the YAML file are meant to be pure date data in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) format. They are not printed in PDF in that format anyway. They are converted into normal English strings (or into any language, by updating [`locale_catalog`](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#locale_catalog-field) field in the YAML file).

Allowing two digits would be a problem because people still use the 1980s and 1990s in their CVs.

**Comment by nf9k** [comment]
Again, something else I discovered after rendering and playing around a bit.

Thank you, again!
